### DISCUSSION TITLE
Discrepancy between selected and installed release

### DISCUSSION THREAD

**Discussion by Reamer** [okd-project/okd#650] [Q&A]
I have noticed a discrepancy between the version I have chosen and the cluster installed on it.

Reproducible with the following steps:
```
./oc adm release info quay.io/openshift/okd:4.7.0-0.okd-2021-05-22-050008 --image-for=installer
docker run -v $(pwd):/output -ti --user 0 quay.io/openshift/okd-content@sha256:867eef225e5c084dc464849b96c19745d76f6cc918dd1174c21c73b1e8e5b8fc create ignition-configs
```

The cluster installation completed successfully, but the cluster version was `4.7.0-0.okd-2021-05-25-205857` and not `4.7.0-0.okd-2021-05-22-050008`.

Does anyone have any idea why this is occurring?

**Comment by vrutkovs** [comment]
`version` subcommand says its pointing to latest nightly:
```
/bin/openshift-install unreleased-master-4228-gba1a5fc61dd3be134a24fa6d89ae24012a93385a
built from commit ba1a5fc61dd3be134a24fa6d89ae24012a93385a
release image registry.ci.openshift.org/origin/release:4.7
```
Workaround: `export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=<pull spec of expected release payload>`

@smarterclayton how does pinning to particular release on build work in installer?
